chain/node.go

```diff
@@ -0,0 +1,92 @@
+package chain
```

Comment:
how about merging this into sync package to keep repo struct simple?

Reply:
api and indexer are two independent services, and api also needs the node service.